projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-interface/wpcom-admin-interface.php

```diff
@@ -417,9 +422,31 @@ function wpcom_is_duplicate_views_experiment_enabled() {
 	if ( ( new Host() )->is_wpcom_simple() ) {
 		\ExPlat\assign_current_user( $aa_test_name );
 		$is_enabled = 'treatment' === \ExPlat\assign_current_user( $experiment_name );
+
+		if ( is_automattician() ) {
```

Comment:
Do we need to exclude support sessions from here? I honestly don't know if `is_automattician()` returns true for a12s doing support sessions.

Reply:
Yep, it should work. is_automattician just gets the current user and searches it in an array of usernames. In a support session the username is the the impersonated one.